src/psxyz.c

```diff
@@ -276,6 +276,13 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
 	GMT_Usage (API, 2, "\n%s 3-D Cube: Give <size> as the length of all sides; append q if <size> "
 		"is a quantity in x-units.", GMT_LINE_BULLET);
 
+	GMT_Usage (API, 2, "\n%s 3-D Sphere: Give <size> as sphere diameter [Default unit is cm]. "
+		"Sphere is rendered with radial gradient shading from white at light source to fill color. Modifiers:", GMT_LINE_BULLET);
+	GMT_Usage (API, 3, "+a Set light source azimuth [0, from the right].");
+	GMT_Usage (API, 3, "+e Set light source elevation [90, perpendicular to viewing plane].");
```

Comment:
Does `+a`/`+e` really work? I tried to plot a sphere with `-Sc1c+a90+e10` and expect to see the white light near the north edge, but it gives me:

<img width="1167" height="608" alt="Image" src="https://github.com/user-attachments/assets/55174dcc-bffe-4bd1-916a-c3c26a6bccdf" />